docs/support.md

```diff
@@ -0,0 +1,13 @@
+# Support
+If you are having any issues in installing, configuring, or otherwise issues with KMK, please reach out to us and
+our community here. We do have a bridge to allow communication between the platforms as well, though we recommend
```

Comment:
maybe instead of "to allow communication between the platforms as well", "syncing Matrix channels with Discord channels". Just flows better IMO and doesn't imply anything special needs done on the user's part (a possible interpretation of the other wording)

Reply:
Removed the entire bit of confusion. Let me know if this is acceptable.